homeassistant/components/matter/cover.py

```diff
@@ -123,25 +158,64 @@ def _update_from_device(self) -> None:
                 self._attr_is_opening = False
                 self._attr_is_closing = False
 
-        # current position is inverted in matter (100 is closed, 0 is open)
-        current_cover_position = self.get_matter_attribute_value(
-            clusters.WindowCovering.Attributes.CurrentPositionLiftPercentage
-        )
-        self._attr_current_cover_position = 100 - current_cover_position
-
-        LOGGER.debug(
-            "Current position for %s - raw: %s - corrected: %s",
-            self.entity_id,
-            current_cover_position,
-            self.current_cover_position,
-        )
+        if self._entity_info.endpoint.has_attribute(
+            None, clusters.WindowCovering.Attributes.CurrentPositionLiftPercent100ths
```

Comment:
I'm not 100% sure if this attribute is mandatory so I'm not sure why we should prefer this attribute over the existing one ?

Reply:
In Matter Application Cluster Specification Version 1.0, `CurrentPositionLiftPercent100ths` attribute conformance is `LF & PA_LF` while `CurrentPositionLiftPercentage` attribute conformance is `[LF & PA_LF]` (Same for tilt attributes) and as far as I know `[ ]` means optional.